psl/schema-ast/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2021"
 [dependencies]
 diagnostics = { path = "../diagnostics" }
 
-pest = "2.1.3"
-pest_derive = "2.1.0"
+pest = "2.7.10"
+pest_derive = "2.7.10"
```

Comment:
This bump is required because old version generated empty comments, which clippy is now allergic to